### PR TITLE
Code Review: Add XCode 13.4 build support  (#44096)

### PR DESCRIPTION
**Pull request by sketch-joseluis**
Connect sketch-hq/sketch#44096

Created by `bo pr`.

### CODE REVIEW

**Review by SheffieldKevin — APPROVED** [review]

**Comment by SheffieldKevin** [comment]
@sketch-joseluis I've approved both of the submodule PRs.

**Comment by sketch-joseluis** [comment]
> @sketch-joseluis I've approved both of the submodule PRs.

Thanks!! I'll merge them now, and prepare a manual merge later today when Mergebot is not very active! :D

**Review by whakkee — APPROVED** [review]
approving as there were changes after Kevin's approval

**Pull request merged**
